### PR TITLE
Documenting how to configure system resources for eventing

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

## Proposed Changes 

- adding doc on how to configure the system resources for knative eventing components, using the operator

### CODE REVIEW

**Comment by matzew** [comment]
/hold

needs to be merged after: https://github.com/knative/eventing/pull/3821

File: docs/install/operator/configuring-eventing-cr.md [hunk 1]
```diff
@@ -243,3 +244,29 @@ metadata:
 spec:
   defaultBrokerClass: MTChannelBasedBroker
 ```
+
+## System Resource Settings
+
+The operator custom resource allows you allows you to configure system resources for the Knative system containers.
```
Comment (abrennan89) [inline docs/install/operator/configuring-eventing-cr.md:250]: Where does someone find or edit this CR? Is `operator` the literal name of the CR or is this referring to KnativeEventing?

File: docs/install/operator/configuring-eventing-cr.md [hunk 2]
```diff
@@ -243,3 +244,29 @@ metadata:
...
+Requests and limits can be configured for the following containers: `eventing-controller`, `eventing-webhook`, `imc-controller`, `imc-dispatcher`, `mt-broker-ingress`, `mt-broker-ingress` and `mt-broker-controller`.
```
Comment (abrennan89) [inline docs/install/operator/configuring-eventing-cr.md:251]:
```suggestion
Requests and limits can be configured for the following containers:

- eventing-controller
- eventing-webhook
- imc-controller
- imc-dispatcher
- mt-broker-ingress
- mt-broker-ingress
- mt-broker-controller
```

Comment (abrennan89) [reply docs/install/operator/configuring-eventing-cr.md:251]: If the requests / limits are set in KnativeEventing, are they automatically applied for all of these containers, or how does that work? Do these need to be configured individually and if so do we make it clear how to do that somewhere?

File: docs/install/operator/configuring-eventing-cr.md [hunk 3]
```diff
@@ -243,3 +244,29 @@ metadata:
...
+
+To override resource settings for a specific container, create an entry in the `spec.resources` list with the container name and the [Kubernetes resource settings](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container).
+
+For example, the following KnativeEventing resource configures the `eventing-webhook` to request 0.3 CPU and 100MB of RAM, and sets hard limits of 1 CPU, 250MB RAM, and 4GB of local storage:
```
Comment (abrennan89) [inline docs/install/operator/configuring-eventing-cr.md:255]:
```suggestion
For example, the following KnativeEventing resource configures the `eventing-webhook` container to request 0.3 CPU and 100MB of RAM, and sets hard limits of 1 CPU, 250MB RAM, and 4GB of local storage:
```

File: docs/install/operator/configuring-eventing-cr.md (hunk 1 above)
Comment (mpetason) [inline docs/install/operator/configuring-eventing-cr.md:250]:
```suggestion
The operator custom resource allows you to configure system resources for the Knative system containers.
```

**Comment by matzew** [comment]
I need to update this ... will do it soon!

**Comment by mpetason** [comment]
@matzew thanks for the work on this so far! I wanted to reach out to see if there are any updates, or if you need help with the changes. It looks like the other issue that this was waiting on was closed instead of merged. Does this require the other PR, or can it be updated/merged by itself?

**Comment by matzew** [comment]
@googlebot I fixed it.

**Comment by matzew** [comment]
reabsed b/c of CLA bot fu .... :man_shrugging:

**Comment by matzew** [comment]
@googlebot I fixed it.

**Review by abrennan89 — APPROVED** [review]
/lgtm
/approve

Thanks @matzew!

**Pull request merged**